tdms/include/argument_parser.h

```diff
@@ -131,6 +131,8 @@ class ArgumentParser {
 private:
   /** Prints the help message (all options).  */
   static void print_help_message();
+  /** Prints this software version. */
```

Comment:
```suggestion
  /** Prints the version of the compiled executable. */
```

There is no subject for "this" to refer to.